doc/source/user_guide/meshing_workflow/new_meshing_workflows.rst

```diff
@@ -1,9 +1,9 @@
 .. _ref_user_guide_new_meshing_workflows:
 
-Using meshing workflows
-=======================
-Fluent meshing guided workflows are designed to make it easier to generate a
-volume mesh starting from CAD geometries. You can use PyFluent to access the guided workflows.
+Enhanced meshing workflow
+=========================
+Provides sample usage of ``watertight`` and ``fault-tolerant`` meshing by using
+the enhanced meshing workflow interface.
```

Comment:
This is a fragment and we don't need to say this here. A slight adjustment to the existing text was fine. Simply, the final sentence could read: "You can use PyFluent to access the new, enhanced meshing workflows."